src/Features/LanguageServer/Protocol/Features/TodoComments/TodoCommentOptionsStorage.cs

```diff
@@ -16,10 +16,17 @@ internal static class TodoCommentOptionsStorage
             TodoCommentOptions.Default.TokenList,
             new RoamingProfileStorageLocation("Microsoft.VisualStudio.ErrorListPkg.Shims.TaskListOptions.CommentTokens"));
 
+        public static readonly Option2<bool> ComputeTodoCommentsForClosedFiles = new(
```

Comment:
do we need to add an option for this, considering we didn't used to have one (and we're splitting them out into a separate request)?

Reply:
yeah, this will likely go away once we have a distinct request here.  this is an internal option only as well, just to control tests to make things not a PITA :)